### PR TITLE
update builtins

### PR DESCRIPTION
**Pull request by 6gx7iycn53ioq2e8apk1j1ypwov4giui**
adds the following builtins:
- inspector - [v8.0.0](https://nodejs.org/en/blog/release/v8.0.0/)
- trace_events - [v10.0.0](https://nodejs.org/en/blog/release/v10.0.0/)
- worker_thread - [v10.5.0](https://nodejs.org/en/blog/release/v10.5.0/)
- wasi - [v12.16.0](https://nodejs.org/en/blog/release/v12.16.0/)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -43,9 +43,13 @@ module.exports = function (version) {
   if (semver.lt(version, '6.0.0')) coreModules.push('freelist')
   if (semver.gte(version, '1.0.0')) coreModules.push('v8')
   if (semver.gte(version, '1.1.0')) coreModules.push('process')
+  if (semver.gte(version, '8.0.0')) coreModules.push('inspector')
   if (semver.gte(version, '8.1.0')) coreModules.push('async_hooks')
   if (semver.gte(version, '8.4.0')) coreModules.push('http2')
   if (semver.gte(version, '8.5.0')) coreModules.push('perf_hooks')
-
+  if (semver.gte(version, '10.0.0')) coreModules.push('trace_events')
+  if (semver.gte(version, '10.5.0')) coreModules.push('worker_threads')
```
Comment (juliangruber) [inline index.js:51]: This looks correct but isn't available in my test 🤔 

```bash
julian@Julians-MBP server % node -v
v10.5.0
julian@Julians-MBP server % node
> require('worker_threads')
Error: Cannot find module 'worker_threads'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:594:15)
    at Function.Module._load (internal/modules/cjs/loader.js:520:25)
    at Module.require (internal/modules/cjs/loader.js:650:17)
    at require (internal/modules/cjs/helpers.js:20:18)
```

Comment (juliangruber) [reply index.js:51]: Ah! They require `--experimental-worker`

**Review by juliangruber — APPROVED** [review]

**Review by juliangruber — CHANGES_REQUESTED** [review]
Could you update the one failing test please?

**Comment by 6gx7iycn53ioq2e8apk1j1ypwov4giui** [comment]
> Could you update the one failing test please?

Yes, of course.

**Comment by 6gx7iycn53ioq2e8apk1j1ypwov4giui** [comment]
@juliangruber I couldn't find a way to pass those arguments to a specific node version in travis.

**Comment by juliangruber** [comment]
Awesome work! https://github.com/juliangruber/builtins/releases/tag/v4.0.0

**Pull request merged**